Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -156,6 +157,11 @@ public function subgraphForNode(Node $node): ContentSubgraphInterface
         return $this->contentRepositoryRegistry->subgraphForNode($node);
     }
 
+    public function isDisabled(Node $node): bool
+    {
+        return $node->tags->contain(SubtreeTag::disabled());
+    }
```

Comment:
the addition of this utility might also be better of in a separate pr where we can explain its usage and think of migrations